client/my-sites/stats/stats-chart-tabs/index.jsx

```diff
@@ -98,47 +95,40 @@ class StatModuleChartTabs extends Component {
 	makeQuery = () => this.props.requestChartCounts( this.props.query );
 
 	render() {
-		const { isActiveTabLoading, className, hideLegend, showChartHeader = false } = this.props;
-		const classes = [
-			'is-chart-tabs',
+		const {
+			isActiveTabLoading,
 			className,
-			{
-				'is-loading': isActiveTabLoading,
-				'has-less-than-three-bars': this.props.chartData.length < 3,
-			},
-		];
-		/* pass bars count as `key` to disable transitions between tabs with different column count */
+			chartData,
+			barClick,
+			counts,
+			charts,
+			children,
+			chartTab,
+			queryDate,
+		} = this.props;
+
 		return (
-			<div className={ clsx( ...classes ) }>
-				{ showChartHeader && (
-					<ChartHeader
-						showLegend={ ! hideLegend }
-						activeLegend={ this.props.activeLegend }
-						activeTab={ this.props.activeTab }
-						availableLegend={ this.props.availableLegend }
-						onLegendClick={ this.onLegendClick }
-						charts={ this.props.charts }
-					></ChartHeader>
-				) }
-
-				<StatsModulePlaceholder className="is-chart" isLoading={ isActiveTabLoading } />
-				<Chart barClick={ this.props.barClick } data={ this.props.chartData } minBarWidth={ 35 }>
-					<StatsEmptyState />
-				</Chart>
-				<StatTabs
-					data={ this.props.counts }
-					tabs={ this.props.charts }
-					switchTab={ this.props.switchTab }
-					selectedTab={ this.props.chartTab }
-					activeIndex={ this.props.queryDate }
-					activeKey="period"
-					aggregate={ isNewDateFilteringEnabled }
-				/>
-			</div>
+			<StatsChartBase
```

Comment:
Yeah, you are right - StatModuleChartTabs is just passing through children to StatsChartBase without adding any value. Since StatsChartBase already handles the slot logic, this wrapper is redundant and adds unnecessary complexity. It is an artifact of some of my previous methods and processes 😄  Thanks for pointing it out!